src/xcode/ENA/ENA/Source/Services/Exposure Submission/ExposureSubmissionService.swift

```diff
@@ -220,13 +217,13 @@ class ENAExposureSubmissionService: ExposureSubmissionService {
 					self._fakeVerificationAndSubmissionServerRequest()
 				case .success(let token):
 					// because this block is only called in QR submission
-					self.updateStoreWithQRSubmissionSelected()
+					Analytics.collect(.testResultMetadata(.registerNewTestMetadata(Date(), token)))
+					Analytics.collect(.keySubmissionMetadata(.submittedWithTeletan(true)))
```

Comment:
This value will be false. This block is only called in QR submission.

Reply:
Good point. I will set the value to `false`